lib/loaders/css-extract.js

```diff
@@ -22,13 +23,17 @@ module.exports = {
      */
     prependLoaders(webpackConfig, loaders) {
         if (!webpackConfig.extractCss) {
+
+            const options = {
+                sourceMap: webpackConfig.useSourceMaps,
+            };
+
```

Comment:
NOW this can be removed - #710 is merged. Sorry for the back and forth.

Reply:
I've done that.